exonum/src/storage/proof_map_index/proof.rs

```diff
@@ -693,6 +680,19 @@ where
                                 .create();
                         }
                     }
+                } else {
+                    // Both children of `branch` do not fit
```

Comment:
Missing dot. And comment itself isn't helpful enough, I think.

Reply:
Please do not remove any comments from proof_map implementation. :upside_down_face: I doubt anyone understands this code completely.

Reply:
I want this comment to be more detailed. Maybe it'll be a good idea to create an issue to document this code well?

Reply:
I didn't change anything in the code, just make changes of if-else statement according to if_not_else lint.

Reply:
Ah, I didn't notice that. Then I'll hide this remark.